### PR TITLE
Fix issue with API response not containing all tags for runtime version update

### PR DESCRIPTION
**Pull request by vatsrahul1001**
We noticed that the [action](https://github.com/astronomer/astronomer-providers/actions/runs/8746284419) job to update the runtime version in the integration test deployment created a [PR](https://github.com/astronomer/astronomer-providers/pull/1513) in which the version was not the latest. After debugging, we found out that the response from the API [call](https://github.com/astronomer/astronomer-providers/blob/afdbace9f41dab4fe7159926505604a0e18735b4/.github/scripts/get_latest_runtime_image_tag.py#L14) does not contain all the tags. This PR addresses the issue after adding a limit query parameter

### CODE REVIEW

**Review by Lee-W — APPROVED** [review]

File: .github/scripts/get_latest_runtime_image_tag.py [hunk 1]
```diff
@@ -25,7 +26,10 @@ def get_latest_tag(repository: str) -> str:
         except ValueError:
             continue
     if valid_tags:
+        print(valid_tags)
```
Comment (pankajkoti) [inline .github/scripts/get_latest_runtime_image_tag.py:29]: Can we remove this? 
```suggestion
```

File: .github/scripts/get_latest_runtime_image_tag.py [hunk 2]
```diff
@@ -25,7 +26,10 @@ def get_latest_tag(repository: str) -> str:
...
         latest_tag = max(valid_tags)
+
+        # print(latest_tag)
```
Comment (pankajkoti) [inline .github/scripts/get_latest_runtime_image_tag.py:32]:
```suggestion
```

File: .github/scripts/get_latest_runtime_image_tag.py [hunk 3]
```diff
@@ -10,10 +10,11 @@
 
 def get_latest_tag(repository: str) -> str:
     """Get the latest semantic version tag from a Quay.io repository."""
-    url = f"https://quay.io/api/v1/repository/{repository}/tag/"
+    url = f"https://quay.io/api/v1/repository/{repository}/tag/?limit=1000"
     response = requests.get(url)
     response.raise_for_status()
     data = response.json()
+    print("Response data:", data)
```
Comment (pankajkoti) [inline .github/scripts/get_latest_runtime_image_tag.py:17]:
```suggestion
```

File: .github/scripts/get_latest_runtime_image_tag.py (hunk 1 above)
Comment (Lee-W) [reply .github/scripts/get_latest_runtime_image_tag.py:29]: I thought this line was for easy debugging?

**Pull request merged**